mkdocs.yml

```diff
@@ -76,6 +76,8 @@ nav:
           - guides/input-events.md
           - guides/troubleshooting.md
           - guides/capability-sets.md
+          - guides/elements-lookup-troubleshooting.md
```

Comment:
There are currently 3 articles for troubleshooting (`troubleshooting`, `wda-slowness` and `elements-lookup-troubleshooting`). I think it's worth adding a new subgroup in this hierarchy under `Guides` for these 3 files - I guess it can also be named just `Troubleshooting`, and placed between `Driver Actions` and `Other`.

Reply:
Maybe it's even worth adding them under a top-level category, though this would also require configuring the redirects.

Reply:
done